AppWidget/glance-widget/README.md

```diff
@@ -0,0 +1,104 @@
+
+# AppWidget sample for Glance
+
+This sample demonstrates how to build widgets using Glance, a new API powered by Compose runtime.
+
+## Build a first widget
+
+You need to prepare at least the following components to build a widget using Glance.
+
+1. AppWidgetProviderInfo metadata
+
+[`first_glance_widget_info.xml`](src/main/res/xml/first_glance_widget_info.xml)
+```xml
+<appwidget-provider xmlns:android="http://schemas.android.com/apk/res/android"
+    android:description="@string/app_widget_description"
+    android:minWidth="180dp"
+    android:minHeight="50dp"
+    android:initialLayout="@layout/widget_empty"
+    android:previewImage="@drawable/first_glance_widget_preview"
+    android:previewLayout="@layout/widget_first_glance_preview"
+    android:resizeMode="horizontal|vertical"
+    android:targetCellWidth="3"
+    android:targetCellHeight="1"
+    android:widgetCategory="home_screen" />
+```
+
+Define the metadata as an XML file in a same format of how you can create a widget without Glance
+and have a reference of the metadata from the AndroidManifest.
+
+```xml
+<receiver
+    android:name=".FirstGlanceWidgetReceiver"
+    android:exported="false">
```

Comment:
As discussed above, I created a boolean value that's true for APIs Glance App Widget is fully supported. So we should encourage developers to add:

```
android:enabled="@bool/glance_appwidget_available"
```

Reply:
Done